### PR TITLE
Readme, Contributors, Changelog updates for v0.3.0

### PR DESCRIPTION
**Pull request by adam-fowler**
- Update Readme task and add command section
- Add section for 0.3.0 for changeling

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unknown]
+## 0.3.0
 
 ### Added
 - Function documentation comment completion. Type "///" on line above function to activate. 
+- Right click menu to package dependency view. Entries include
```
Comment (0xTim) [inline CHANGELOG.md:11]:
```suggestion
- Right click menu to see package dependency view. Entries include:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
...
+  - Use Local Version: Use local version of package dependency
```
Comment (0xTim) [inline CHANGELOG.md:12]: Nit consistency
```suggestion
  - Use Local Version: Use local version of package dependency.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
...
+  - Add To Workspace: Add a locally edited dependency to your VSCode Workspace
```
Comment (0xTim) [inline CHANGELOG.md:13]:
```suggestion
  - Add To Workspace: Add a locally edited dependency to your VSCode Workspace.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
...
+  - Revert To Original Version: Revert locally edited dependency to the version in the Package.swift
```
Comment (0xTim) [inline CHANGELOG.md:14]:
```suggestion
  - Revert To Original Version: Revert locally edited dependency to the version in the Package.swift.
```

**Review by 0xTim — APPROVED** [review]
Minor nits otherwise LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (adam-fowler) [reply CHANGELOG.md:11]: The right click menu isn't to see the dependency view. It is to add options for dependencies in it eg use local version, view repository

Comment (0xTim) [reply CHANGELOG.md:11]: Should this be for or add instead then? The sentence confused me

Comment (adam-fowler) [reply CHANGELOG.md:11]: I'll reword

"Package dependency view has new right click menu. Menu entries include."

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,10 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unknown]
+## 0.3.0
 
 ### Added
 - Function documentation comment completion. Type "///" on line above function to activate. 
+- Package dependency view has new right click menu. Menu entries include
```
Comment (0xTim) [inline CHANGELOG.md:11]: Grammar nit

```suggestion
- Package dependency view has new right click menu. Menu entries include:
```

**Review by 0xTim — APPROVED** [review]

**Pull request merged**